app/package.json

```diff
@@ -38,13 +39,14 @@
   "devDependencies": {
     "@testing-library/cypress": "^7.0.4",
     "assert": "^2.0.0",
-    "cypress": "^6.8.0",
+    "cypress": "^9.0.0",
     "fetch-mock": "^6.5.1",
     "fetch-readablestream": "^0.2.0",
     "sinon": "^7.3.2"
   },
   "resolutions": {
     "lodash": ">=4.17.21",
-    "ansi-regex": "<6.0.0"
+    "ansi-regex": "<6.0.0",
+    "cypress/**/json-schema": ">=0.4.0"
```

Comment:
for cypress it turns out I needed to override the json-schema package version as the latest version seems to still recursively depend on the vulnerable one